.github/actions/javascript/getArtifactInfo/index.js

```diff
@@ -16000,7 +16024,7 @@ module.exports = JSON.parse('[[[0,44],"disallowed_STD3_valid"],[[45,46],"valid"]
 /******/ 	// startup
 /******/ 	// Load entry module and return exports
 /******/ 	// This entry module is referenced by other modules so it can't be inlined
-/******/ 	var __webpack_exports__ = __nccwpck_require__(1307);
+/******/ 	var __webpack_exports__ = __nccwpck_require__(7750);
```

Comment:
This file is auto-generated so I don't know the exact reason behind that change, but my guess is this might be due to `module.exports` changing to `export default`. Wdyt?

Reply:
Ahh I see.